src/main.rs

```diff
@@ -170,37 +186,48 @@ fn make_request(args: &Swww) -> Result<Option<Request>, String> {
 
 fn make_img_request(
     img: &cli::Img,
-    img_raw: image::RgbImage,
+    img_raw: Option<image::RgbImage>,
```

Comment:
I feel like making this an `Option` and checking that it is not `None` for `CliImage::Path` below is kinda unfortunate because in the future this can lead to errors on misuse. Maybe it would be better to create the image in `make_request` (line 171) beforehand (move lines 227-229) and leave it without an `Option`?

Reply:
moved the img gen to `make_request` in c47ba76ffb2632d3c6b7b579f6b36e85513c6222

Reply:
well after trying to fix conflicts i changed it to a better way, instead of sending Option, we wont send imgbuf at all and make it in `make_img_request` directly